components/resources/permissions_strings.grdp

```diff
@@ -21,4 +21,10 @@
   <message name="IDS_PERMISSIONS_BUBBLE_SITE_PERMISSION_LINK" desc="Site permission link in the footnote description.">
     site permission
   </message>
+  <message name="IDS_OPEN_AI_CHAT_PERMISSION_FRAGMENT" desc="Label for Leo AI chat permission prompt.">
+    Allow opening Leo in Brave?
+  </message>
+  <message name="IDS_AI_CHAT_INFOBAR_TEXT" desc="Label for Leo AI chat permission prompt (Android).">
+    Allow <ph name="URL">$1<ex>search.brave.com</ex></ph> to open Leo in Brave?
```

Comment:
"Allow search.brave.com to open related Leo AI conversations in Brave?"?

Reply:
"Allow search.brave.com to open Leo AI conversations in Brave?" in https://github.com/brave/brave-core/commit/ab4ea07f080379399d945e2357de36394ecf6fc0